retroarch_data.h

```diff
@@ -2373,6 +2373,7 @@ struct aspect_ratio_elem aspectratio_lut[ASPECT_RATIO_END] = {
    { 0.0f,            "Config"        },
    { 1.0f,            "Square pixel"  },
    { 1.0f,            "Core provided" },
+   { 1.3333f,         "Full" },
```

Comment:
Thanks for making the changes! Just one last thing - the order of these entries needs to be updated to match the order of `enum aspect_ratio`.

If you could do that, and then squash your commits, we'll be ready to go :)